### DISCUSSION TITLE
Throw exception 'ModuleNotFoundException: [onfiguration-discovery] missing' while executing bin/startup.sh

### DISCUSSION THREAD

**Discussion by yixiubest** [apache/skywalking#9853] [Q&A] [answered]
first------./bin/startup.sh
then show logs:
![image](https://user-images.githubusercontent.com/38375081/198173843-f5f3e2ef-815a-48ec-8dd4-ad222e4478b8.png)

So why?

**Comment by wu-sheng** [comment]
You main use some old version application.yml to boot new backend server.

**Comment by kezhenxu94** [comment, marked as answer]
@Gitttttttt check you application.yml, you might have typo there. Search `onfiguration-discovery` and modify it to `configuration-discovery`

`org.apache.skywalking.oap.server.library.module.ModuleNotFoundException:(onfiguration-discovery]missing.`

**Reply by yixiubest** [reply]
yeah,i had found it,thank u as well.
And i have another question: my skywalking-apm version is v9.2.0, but there's no agent folder, had something changed for the version?
![image](https://user-images.githubusercontent.com/38375081/198179736-b03b07ea-394d-4ce3-9472-9009572a803d.png)

**Reply by yixiubest** [reply]
Do i need to download the java agent package separately？

**Reply by wu-sheng** [reply]
https://skywalking.apache.org/docs/#JavaAgent

It is a separate subproject for a long time.

**Reply by yixiubest** [reply]
ok,I have founded that，thks

**Comment by yixiubest** [comment]
it was my mistake while editting yaml,i'm so foolish...
